modules/apps/remote-app/remote-app-service/src/main/java/com/liferay/remote/app/service/impl/RemoteAppEntryLocalServiceImpl.java

```diff
@@ -132,14 +136,19 @@ public RemoteAppEntry addCustomElementRemoteAppEntry(
 		remoteAppEntry.setProperties(properties);
 		remoteAppEntry.setSourceCodeURL(sourceCodeURL);
 		remoteAppEntry.setType(RemoteAppConstants.TYPE_CUSTOM_ELEMENT);
+		remoteAppEntry.setStatus(WorkflowConstants.STATUS_DRAFT);
+		remoteAppEntry.setStatusByUserId(userId);
+		remoteAppEntry.setStatusDate(new Date());
 
 		remoteAppEntry = remoteAppEntryPersistence.update(remoteAppEntry);
 
 		_addResources(remoteAppEntry);
 
 		remoteAppEntryLocalService.deployRemoteAppEntry(remoteAppEntry);
 
-		return remoteAppEntry;
+		// Workflow
```

Comment:
Is this SF compliant? :thinking:

Reply:
Yes. Why does this feel weird to you? One thing it is that the SF doesn't comply and another is that it may be wrong

Reply:
No, simply because comments are not very common. But I've seen some like these in other places, that's why I asked :+1:

Reply:
It's true that SF doesn't complain about comments, so I didn't explained myself correctly.